python/dask_cudf/dask_cudf/io/parquet.py

```diff
@@ -332,12 +332,13 @@ def set_object_dtypes_from_pa_schema(df, schema):
     # pyarrow schema.
     if schema:
         for col_name, col in df._data.items():
-            if col_name in schema.names and isinstance(
-                col, cudf.core.column.StringColumn
+            typ = schema.field(col_name).type
```

Comment:
Any particular reason we don't opt to call `cudf_dtype_from_pa_type` up here, and avoid having to import `pyarrow`?

```suggestion
            typ = cudf_dtype_from_pa_type(schema.field(col_name).type)
```